VAMobile/src/screens/HomeScreen/ProfileScreen/PersonalInformationScreen/PersonalInformationScreen.tsx

```diff
@@ -8,28 +8,31 @@ import { Box, BoxProps, ErrorComponent, FeatureLandingTemplate, LargeNavButton,
 import { GenderIdentityOptions, UserDemographics } from 'api/types/DemographicsData'
 import { HomeStackParamList } from 'screens/HomeScreen/HomeStackScreens'
 import { NAMESPACE } from 'constants/namespaces'
-import { PersonalInformationState } from 'store/slices/personalInformationSlice'
-import { RootState } from 'store'
+import { PersonalInformationData } from 'api/types/PersonalInformationData'
 import { ScreenIDTypesConstants } from 'store/api/types'
-import { UserDataProfile } from 'store/api/types'
 import { a11yLabelVA } from 'utils/a11yLabel'
 import { featureEnabled } from 'utils/remoteConfig'
 import { formatDateMMMMDDYYYY, stringToTitleCase } from 'utils/formattingUtils'
+import { getAllFieldsThatExist } from 'utils/common'
 import { registerReviewEvent } from 'utils/inAppReviews'
 import { useDemographics } from 'api/demographics/getDemographics'
 import { useError, useRouteNavigation, useTheme } from 'utils/hooks'
 import { useGenderIdentityOptions } from 'api/demographics/getGenderIdentityOptions'
-import { useSelector } from 'react-redux'
+import { usePersonalInformation } from 'api/personalInformation/getPersonalInformation'
 
-export const getBirthDate = (profile: UserDataProfile | undefined, t: TFunction): string => {
-  if (profile && profile.birthDate) {
-    const formattedBirthDate = formatDateMMMMDDYYYY(profile.birthDate)
+export const getBirthDate = (birthDate: string | undefined, t: TFunction): string => {
+  if (birthDate) {
+    const formattedBirthDate = formatDateMMMMDDYYYY(birthDate)
     return t('dynamicField', { field: formattedBirthDate })
   } else {
     return t('personalInformation.informationNotAvailable')
   }
 }
 
+export const getFullName = (personalInfo: PersonalInformationData | undefined): string => {
+  return personalInfo ? getAllFieldsThatExist([personalInfo.firstName, personalInfo.middleName, personalInfo.lastName]).join(' ').trim() || '' : ''
+}
+
```

Comment:
Instead of having this be a function, you can set `fullName` in the `getPersonalInformation` query function, similar to how we are with the formatted phone numbers in the contact info [query function](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/6521804f5f754beb015253f92c1817587015f82f/VAMobile/src/api/contactInformation/getContactInformation.tsx#L16-L21). This would avoid needing to import a function for all the screens where the full name is needed.

Reply:
updated, also did the same for birthdate